### PR TITLE
Config Global Resources - Mocking Support

### PR DESCRIPTION
**Pull request by wey-chiang**
### Background

The tests were commented out pending mocking support. This PR adds support for unit test mocking for these uncommented tests.

### Changes

* Replaced commented out tests with mocking to provide examples

### Testing

* `panther_analysis_tool test --path=./`

### CODE REVIEW

File: aws_config_policies/aws_config_global_resources.yml [hunk 1]
```diff
@@ -31,42 +31,52 @@ Tests:
...
-#        "ResourceId": "123456789012::AWS.Config.Recorder.Meta"
-#      }
+  -
+    Name: Global Resources Are Enabled
+    ExpectedResult: true
+    Mocks:
+      -
+        {
+          "objectName": "resource_lookup",
+          "returnValue": "True"
```
Comment (nhakmiller) [inline aws_config_policies/aws_config_global_resources.yml:41]: "{"RecordingGroup": {"IncludeGlobalResourceTypes": true}, "Status": {"Recording": true}}"  or  something similar.

File: aws_config_policies/aws_config_global_resources.py [hunk 2]
```diff
@@ -15,6 +16,8 @@ def policy(resource):
 
     for recorder_name in resource.get("Recorders", []):
         recorder = resource_lookup(recorder_name)
+        if isinstance(recorder, str):
+            return literal_eval(recorder)
```
[the remark this answers is not in the document]
Comment (wey-chiang) [reply aws_config_policies/aws_config_global_resources.py:20]: Makes sense, I just recently wrote up a walkthrough for this in the docs so I'll default to that.

**Comment by wey-chiang** [comment]
Tested that the mocks actually work on this example data by flipping the boolean for `Recording` to verify the tests fail when flipped (which should only happen if the logic and mocking happens correctly)

**Review by nhakmiller — APPROVED** [review]
🚀

**Pull request merged**